pkg/services/accesscontrol/seeder/roles.go

```diff
@@ -0,0 +1,88 @@
+package seeder
+
+import "github.com/grafana/grafana/pkg/services/accesscontrol"
+
+var builtInRoles = []accesscontrol.RoleDTO{
```

Comment:
I think it's worth to have a discussion about renaming built-in roles to predefined roles (I will be fine with also renaming predefined roles to built-in roles). But after adding these two new roles, I can see that logically built-in roles and predefined roles are essentially the same concept. cc @alexanderzobnin @gamab